package.json

```diff
@@ -5,17 +5,25 @@
   "scripts": {
     "ng": "ng",
     "cpy": "cpy",
+    "start-test": "start-server-and-test",
     "start": "ng serve",
     "build": "ng-packagr -p src/lib/picker/ng-package.json",
     "postbuild": "cpy README.md LICENSE dist",
+    "build:integration:prod": "ng build --configuration production --no-progress",
+    "build:integration:ghpages": "ng build --configuration ghpages --no-progress",
+    "serve:integration:static": "serve dist -s -l 4200 --cors",
     "build-data": "tsx scripts/build-data.ts",
     "test": "ng test --watch=false",
     "test:watch": "ng test",
     "test:ci": "ng test --watch=false --code-coverage --no-progress --browsers=ChromeHeadlessCustom",
     "link": "cd dist && npm link",
     "lint": "ng lint",
     "lint:fix": "ng lint --fix",
-    "ghpages": "ng build --configuration production --no-progress"
```

Comment:
this doesn't actually deploy to github pages anymore, it deploys the demo to vercel i just never changed the name

Reply:
should we change it to smth like `build:vercel`?

Reply:
i guess 🤷

Reply:
updated